zulipterminal/helper.py

```diff
@@ -410,3 +418,21 @@ def canonicalize_color(color: str) -> str:
         return color.lower()
     else:
         raise ValueError('Unknown format for color "{}"'.format(color))
+
+
+@asynch
+def notify(title: str, text: str) -> None:
```

Comment:
As per another comment, this might be clearer as `html_text`; this also ensures that the `text` variable is clearly separate.